darts/explainability/shap_explainer.py

```diff
@@ -0,0 +1,690 @@
+                    )
+                shap_values_dict[h] = tmp
+
+            shap_values_list.append(shap_values_dict)
+
+        if len(shap_values_list) == 1:
+            shap_values_list = shap_values_list[0]
+
+        return ExplainabilityResult(shap_values_list)
+
+    def summary_plot(
+        self,
+        horizons: Optional[Sequence[int]] = None,
+        target_names: Optional[Sequence[str]] = None,
+        nb_samples: Optional[int] = None,
+        plot_type: Optional[str] = "dot",
+        **kwargs,
+    ):
+        """
+        Display a shap plot summary per (target, horizon)).
+        We here reuse the initial background data as foreground (potentially sampled) to give a general importance
+        plot for each feature.
+        If no target names and/or no horizons are provided, we plot all summary plots.
+
+        Parameters
+        ----------
+        horizons
+            Optionally, a list of integers representing which points/steps in the future we want to explain,
+            starting from the first prediction step at 0. Currently, only forecasting models are supported which
+            provide an `output_chunk_length` parameter. `horizons` must not be larger than `output_chunk_length`.
+        target_names
+            Optionally, a list of strings with the target components we want to explain.
+        nb_samples
+            Optionally, an integer for sampling the foreground series (based on the backgound),
+            for the sake of performance.
+        plot_type
+            Optionally, specify which of the propres shap library plot type to use. Can be one of
+            ``'dot', 'bar', 'violin'``.
+
+        """
+
+        horizons, target_names = self._check_horizons_and_targets(
+            horizons, target_names
+        )
+
+        if nb_samples:
+            foreground_X_sampled = shap.utils.sample(
+                self.explainers.background_X, nb_samples
+            )
+        else:
+            foreground_X_sampled = self.explainers.background_X
+
+        shaps_ = self.explainers.shap_explanations(
+            foreground_X_sampled, horizons, target_names
+        )
+
+        for t in target_names:
+            for h in horizons:
+                plt.title("Target: `{}` - Horizon: {}".format(t, "t+" + str(h)))
+                shap.summary_plot(
+                    shaps_[h][t],
+                    foreground_X_sampled,
+                    plot_type=plot_type,
+                    **kwargs,
+                )
+
+    def force_plot_from_ts(
+        self,
+        horizon: int = None,
+        target_name: str = None,
+        foreground_series: TimeSeries = None,
+        foreground_past_covariates: Optional[TimeSeries] = None,
+        foreground_future_covariates: Optional[TimeSeries] = None,
+        **kwargs,
+    ):
+        """
+        Display a shap force_plot for one target and one horizon.
+        It displays SHAP values of each lag/covariate with an additive force layout.
```

Comment:
(for consistency with the rest of the docs)
```suggestion
        It displays shape values of each lag/covariate with an additive force layout.
```